### PR TITLE
Create Translatathon page

### PR DESCRIPTION
**Pull request by lukassim**
## Description

Create Translatathon landing page

## Related Issue

No related issue, part of a new initiative

### CODE REVIEW

File: src/content/contributing/translation-program/translatathon/index.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+- Prize winners will be subject to KYC before receiving prizes, and any taxes this might incur are the sole responsibility of the participant.
+
+- We are unable to issue prizes to participants from countries that are on sanctions lists (e.g. North Korea, Syria, Iran, and Cuba).
+
+## Frequently asked questions {#FAQ}
+
+<ExpandableCard title="Who can I contact if I need help or have questions, feedback, or ideas?" eventCategory="/translation-program/translatathon" eventName="clicked Who can I contact if I need help or have questions, feedback, or ideas?">
+
+- In Crowdin, you can send a direct message to [Ethereum.org Team](https://crowdin.com/profile/ethdotorg)
```
Comment (wackerow) [inline src/content/contributing/translation-program/translatathon/index.md:153]: If I recall, we decided to stick to html when we're nested inside other components (ie. `ExpandableCard` here)... @minimalsm Can you confirm?

If so, these links should be switched to `<a href="https://link-here">Label</a>`

File: src/content/contributing/translation-program/translatathon/index.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+- On the [ethereum.org Discord](https://www.ethereum.org/discord), you can send a message in the **#translatathon** & **#translate** channels
```
Comment (wackerow) [inline src/content/contributing/translation-program/translatathon/index.md:154]: ..and similar to above, bold items would be switched to `<strong>Text here</strong>`, and italics would be `<em>Text here</em>` as needed

**Comment by corwintines** [comment]
I can push those changes up @wackerow

**Review by corwintines — DISMISSED** [review]

**Review by wackerow — CHANGES_REQUESTED** [review]
This is currently targeting `master` and needs to target `dev`

**Comment by corwintines** [comment]
@wackerow its intentionally pointing to master, needs to go out today for translatathon announcement

**Review by wackerow — APPROVED** [review]
Ok, given this is a hotfix/needs to go out today, approving per https://github.com/ethereum/ethereum-org-website/blob/dev/docs/deploy-process.md

**Pull request merged**